src/lighteval/metrics/llm_as_judge.py

```diff
@@ -235,3 +230,20 @@ def __process_judge_response(self, judgment: str) -> int:
             rating = -1
 
         return rating
+
+    def __call_openai_api(self, prompt):
+        for _ in range(self.API_MAX_RETRY):
+            try:
+                response = self.client.chat.completions.create(
+                    model=self.model,
+                    # seed=self.seed,
+                    # temperature=self.temperature,
+                    messages=prompt,
+                    max_tokens=512,
+                    n=1,
+                )
+                return response.choices[0].message.content
```

Comment:
We might need a check to make sure the content is correct

Reply:
if there is an error, it will be thown that's why we do a try catch